### ISSUE TITLE
blank-lines option for indent rule.

### ISSUE THREAD

**Issue by r-plus** [nicklockwood/SwiftFormat#513]
@nicklockwood Hi, I won't remove white spaces of blank lines by `indent` rule.
What do you think of `--blank-lines` option? (e.g. `remove` (default) and `ignore`)

```swift
class A {
....  <- I won't remove this space
....var a: String?
}
```

**Comment by nicklockwood** [comment]
@r-plus You can use `--disable blankLinesAtStartOfScope` to disable the rule that does this.

**Comment by r-plus** [comment]
@nicklockwood thanks for your response.

sorry, my sample is not good. I have mixed code that have space having blank line and no-space blank line. In this case, `indent` rule will remove all of the spaces from blank lines.

### sample

before format code.

```swift
class A {
....
....var a: String?
[empty-line]
....var b: String?
....
}
```

after the format by `$ swiftformat --rules indent --swiftversion 5.1 test.swift`

```swift
class A {
[empty-line]
....var a: String?
[empty-line]
....var b: String?
[empty-line]
}
```

I'm won't format about blank lines in this case.

**Comment by nicklockwood** [comment]
@r-plus you can add the `--trimwhitespace nonblank-lines` option if you don't want the whitespace in empty lines to be trimmed.

**Comment by r-plus** [comment]
@nicklockwood with that rule and option by following command

`$ swiftformat --rules indent --rules trailingSpace --trimwhitespace nonblank-lines --swiftversion 5.1 test.swift`

file will be formatted to 

```swift
class A {
....
....var a: String?
....  <- changed
....var b: String?
....
}
```

I won't format about blank lines. So i wanting ignore able option for indent rule for blank lines.
How about?

**Comment by nicklockwood** [comment]
Hmm, this is a pretty obscure use-case. Why do you want the blank line preserved in some cases and not others - is it just to avoid noise in the commit?

**Comment by r-plus** [comment]
yes, i'm thinking changes of blank lines are not meaningful change for code readability.

**Comment by ivan-gaydamakin** [comment]
same problem
